TicTacIOiOS/ViewController.swift

```diff
@@ -119,8 +143,9 @@ class ViewController: UIViewController, UIAlertViewDelegate {
         default:
             coord = (-1, -1)
         }
-        
-        socket.emit("playerMove", coord.x, coord.y)
+        if let socket = socket {
+            socket.emit("playerMove", coord.x, coord.y)
```

Comment:
Like here instead of an if let you could just do `socket?.emit("playerMove", coord.x, coord.y)`